ext/random/random.c

```diff
@@ -81,12 +81,8 @@ PHPAPI uint32_t php_random_range32(php_random_algo_with_state engine, uint32_t u
 	const php_random_algo *algo = engine.algo;
 	void *state = engine.state;
 
-	uint32_t result, limit;
+	uint32_t result = 0;
 	size_t total_size = 0;
```

Comment:
```suggestion
    uint32_t result;
    size_t total_size;

	result = 0;
	total_size = 0;
```

Thank you. For `result` and `total_size`, I prefer to keep the declaration separate, again for consistency between the two loops. By keeping the declaration separate the loop at the top can simply be copy-and-pasted without needing to remove the type declaration.

My code suggestions are untested, but should hopefully be correct.